docs/user/metrics/labeled_counters.md

```diff
@@ -18,6 +18,8 @@ stability:
     ...
 ```
 
+> **Note:** removing or changing labels, including their order in the registry file, is permitted. Special care must be taken when touching GeckoView metrics exfiltrated through the Glean SDK, as the index of the labels is used to exfiltrate Gecko data through the Glean SDK.
```

Comment:
Do we want to add something like: 

"Avoid reusing labels that were removed in the past"?

Also, maybe add: "It is best practice to add documentation about removed labels to the `description` field so that analysts will know of their existence and meaning in historical data."  (Or maybe some future version of the probe dictionary will handle that for us automatically -- by shown known, yet decommissioned, labels).

Reply:
Yes! Makes sense, adding it